Cargo.toml

```diff
@@ -3,14 +3,23 @@ name = "quickstitch"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+gui = ["dep:iced"]
+
 [dependencies]
 anyhow = "1.0.86"
-image = "0.25.2"
+image = { version = "0.25.2", features = ["rayon", "jpeg", "png", "webp"] }
 natord = "1.0.9"
 rayon = "1.10.0"
 thiserror = "1.0.63"
-iced = { git = "https://github.com/iced-rs/iced", rev = "043f0302142e46bf1d7ba2015f83261813280fec" }
 itertools = "0.13.0"
 
+# gui deps
+iced = { git = "https://github.com/iced-rs/iced", rev = "043f0302142e46bf1d7ba2015f83261813280fec", optional = true }
```

Comment:
Iced is now an optional dependency, only compiled if you use the `gui` feature. Now, just running tests or benchmarks (and the future cli) will no longer require building it.